www_src/components/action-menu/action-menu.jsx

```diff
@@ -0,0 +1,49 @@
+var React = require('react');
+var classNames = require('classnames');
+var assign = require('react/lib/Object.assign');
+var Link = require('../link/link.jsx');
+
+var Menu = React.createClass({
+  render: function () {
+    return (<div className="action-menu">
+      {this.props.children}
+    </div>);
+  }
+});
+
+var Btn = React.createClass({
+  render: function () {
+    var Tag = this.props.url ? Link : 'button';
+    return (<Tag {...this.props}>
+      <img className="icon" src={this.props.icon} />
+    </Tag>);
+  }
+});
+
+var Primary = React.createClass({
+  render: function () {
+    var className = classNames(this.props.className, {
+      'primary-btn': true,
+      'off': this.props.off
+    });
+    var props = assign({}, this.props, {className});
+    return <Btn {...props} />;
+  }
+});
+
+var Secondary = React.createClass({
```

Comment:
This seems almost identical to Primary. Any way they can be combined or derived from a base class?